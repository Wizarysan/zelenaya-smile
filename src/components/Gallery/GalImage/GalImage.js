import React from 'react';

class GalImage extends React.Component {
  render() {
    return(<img className="gal_nav__photo__image img-responsive center-both"
    src={process.env.PUBLIC_URL+"/galleries/"+this.props.currentId+"/big/"+this.props.image} alt="" />)
  }
}

export default GalImage;
