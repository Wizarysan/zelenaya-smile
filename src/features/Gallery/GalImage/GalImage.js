import React, {PropTypes} from 'react';

class GalImage extends React.Component {
  render() {
    return(<img className="gal_nav__photo__image img-responsive center-both"
    src={process.env.PUBLIC_URL+"/galleries/"+this.props.currentId+"/big/"+this.props.image} />)
  }
}

export default GalImage;
