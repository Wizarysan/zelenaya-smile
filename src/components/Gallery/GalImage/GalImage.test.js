import React from 'react';
import GalImage from './GalImage.js';
import renderer from 'react-test-renderer';

it('renders correctly with test ID and filename', () => {
  const tree = renderer
    .create(<GalImage currentId="1" image="test.jpg"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});