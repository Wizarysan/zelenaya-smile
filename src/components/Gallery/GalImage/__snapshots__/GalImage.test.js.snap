exports[`test renders correctly with test ID and filename 1`] = `
<img
  className="gal_nav__photo__image img-responsive center-both"
  src="/galleries/1/big/test.jpg" />
`;
